run_benchmarks.sh

```diff
@@ -0,0 +1,28 @@
+TMPDIR="${PWD}/bundles"
+if [ -d "$TMPDIR" ]; then
+  # Take action if $DIR exists. #
+  echo "removing ${TMPDIR} before running benchmarks"
+  rm -rf ${TMPDIR}
+fi
+mkdir ${TMPDIR}
+
+DIR="rpm-adapter"
+if [ -d "$DIR" ]; then
+  # Take action if $DIR exists. #
+  echo "remove ${DIR} before running benchmarks"
+  exit 0
+fi
+echo "Currently TMP: ${TMPDIR}"
+#cp /mnt/disk2/projects/rpmtests/bundle1000.tar.gz ${TMPDIR}
+#cp /mnt/disk2/projects/rpmtests/bundle100.tar.gz ${TMPDIR}
+
+git clone https://github.com/artipie/rpm-adapter.git
+cd rpm-adapter
+echo ${PWD}
+mvn package -Pbench
+wget https://artipie.s3.amazonaws.com/rpm-test/bundle100.tar.gz
+tar -xvzf bundle100.tar.gz -C ${TMPDIR}
+wget https://artipie.s3.amazonaws.com/rpm-test/bundle1000.tar.gz
+tar -xvzf bundle1000.tar.gz -C ${TMPDIR}
```

Comment:
@milkysoft we need only one bundle, not both. It could be configurable parameter with `bundle1000` default value

Reply:
@g4s8 , `-b` optional parameter added with latest [commit](https://github.com/artipie/rpm-adapter/pull/386/commits/7607ca7c554416dcdf9bcb7723af6fc9e1f3c09b), which could be manually set to `bundle100` or bundle1000`.